esphome/onju-voice.yaml

```diff
@@ -6,15 +6,17 @@ substitutions:
 
 esphome:
   name: "${name}"
-  friendly_name: "{$friendly_name}"
+  friendly_name: "${friendly_name}"
   comment: "${device_description}"
```

Comment:
Thank you for catching this, but please let's have PRs dedicated to one purpose. Can you please remove the change from here? You can contribute another PR for this change specifically or i can make the change myself later today.